src/components/NotificationCenter/Notification.js

```diff
@@ -1,12 +1,29 @@
 import React from "react";
+import Dropdown, { DropdownItem } from '../library/Dropdown';
 
-export default function Notification({onDismiss, onDeactivate, notification}) {
+export default function Notification({onDismiss, onDeactivate, notification, onSnooze}) {
     return (
       <div className="notification is-warning">
         <button className="delete" onClick={onDismiss} ></button>
         {notification.todo.name} is overdue!
         <div className="notification-buttons">
-          <button className="button is-white is-small" onClick={onDeactivate}>Deactivate</button>
+          <button className="button is-white is-small" onClick={onDeactivate}>Mark as done</button>
+
+          <button className="button snooze is-white is-small">
+            <Dropdown
+              trigger={
+                (<span className="icon is-small">
+                  <i className="clockIcon fas fa-clock"></i>
+                  <i className="snoozeIcon fas fa-angle-down" aria-hidden="true"></i>
+                </span>)
```

Comment:
I think this is not correct in terms of the Bulma way. The correct way would be:
```jsx
<span className="icon is-small">
  <i className="clockIcon fas fa-clock" />
</span>
<span className="icon is-small">
  <i className="snoozeIcon fas fa-angle-down" aria-hidden="true" />
</span>
```
Which in terms of React would not be correct either because React will probably tell you that only **one element should be returned**. You can either wrap things in a real outer element like a span but you can also use a `React fragment`. 
React fragments do not appear in the DOM in the end, React removes them when it compiles the JSX, but it counts as a React element just like any other one so it does not the trigger the error that I mentioned above. React fragments were made exactly for a case like this.
See for more details https://reactjs.org/docs/fragments.html

In the end the correct solution is (sorry for the spoiler):
```jsx
<React.Fragment>
  <span className="icon is-small">
    <i className="clockIcon fas fa-clock" />
  </span>
  <span className="icon is-small">
    <i className="snoozeIcon fas fa-angle-down" aria-hidden="true" />
  </span>
</React.Fragment>
```